.travis.yml

```diff
@@ -1,13 +1,21 @@
 language: go
+go_import_path: github.com/davecgh/go-spew
+
 go:
     - 1.5.4
     - 1.6.3
     - 1.7
+    - 1.8
+    - 1.9
+    - latest
```

Comment:
Where is `latest` coming from?

I believe it should be `master`. Travis documentation at https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use says:

> use `master` to get the latest version from source.

Reply:
I meant to write `tip` here, I've amended the commit and pushed a new version.

Reply:
Ok. But why use `tip` instead of `master` as [latest Travis documentation](https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use) suggests? `tip` was what they used in the past, and continues to work for now, but it may be eventually removed.